runtime/v1/linux/task.go

```diff
@@ -124,7 +124,7 @@ func (t *Task) Start(ctx context.Context) error {
 	t.pid = int(r.Pid)
 	if !hasCgroup {
 		cg, err := cgroups.Load(cgroups.V1, cgroups.PidPath(t.pid))
-		if err != nil {
+		if err != nil && err != cgroups.ErrCgroupDeleted {
 			return err
 		}
 		t.mu.Lock()
```

Comment:
one line below, should you set this to an explicit `nil`?

Reply:
addressed